synedrion/src/tools/secret.rs

```diff
@@ -25,7 +26,10 @@ use crate::{
 /// - Safe serialization/deserialization (down to `serde` API; what happens there we cannot control)
 pub(crate) struct Secret<T: Zeroize>(SecretBox<T>);
 
-impl<T: Zeroize> Secret<T> {
+impl<T> Secret<T>
```

Comment:
Btw, do you use some tool to move the type bounds into the `where` clause? I would like to make it consistent throughout the codebase, but I couldn't find a lint in `clippy` or `rustfmt` setting that would do that.

Reply:
Yes, but it only works as a one-off thing, mapped to a key combo I have to press for each spot in the code. Having a way to do it across the code base would be real nice. Like, when I write code it's often convenient to type the bounds directly, but when reading code it's much easier (for me) to scan for the `where` clause; so automation would work well there for me.